theme/templates/resource-landing-page/references.html

```diff
@@ -160,7 +160,7 @@ <h4>Related Resources
                             {% elif relation.type == "IsReferencedBy" %}
                                 This resource is referenced by:
                             {% else %}
-                                {{ relation.type }}
+                                {{ relation.value }}
```

Comment:
@sblack-usu This change was unintended and there was another commit undoing it. Could you include it in this PR?